File: backend/app/routes/health.py
"""
헬스 체크 라우트
"""
from flask import Blueprint, jsonify
from datetime import datetime

bp = Blueprint('health', __name__)


@bp.route("/")
def health_check():
    """헬스 체크 엔드포인트"""
    return jsonify({
        "status": "ok",
        "message": "KRX Stock Info API Server",
        "timestamp": datetime.now().isoformat()
    })

File: backend-go/internal/server/handlers/health.go
package handlers

import (
	"net/http"
	"time"

	"investment-news-go/internal/httpx"
)

func (h *Handlers) Health(w http.ResponseWriter, r *http.Request) {
	if r.URL.Path != "/" {
		httpx.WriteJSON(w, http.StatusNotFound, map[string]any{
			"error":   "요청한 리소스를 찾을 수 없습니다.",
			"message": "Not Found",
		})
		return
	}

	httpx.WriteJSON(w, http.StatusOK, map[string]any{
		"status":    "ok",
		"message":   "KRX Stock Info API Server",
		"timestamp": time.Now().Format(time.RFC3339),
	})
}
